### ISSUE TITLE
Powertools crashing built in games

### ISSUE THREAD

**Issue by BrooklynTheDogg** [NGnius/PowerTools#40]
Running the latest powertools is crashing some steam deck verified games. Not sure why. Uninstalling powertools fixes the issue. Crashes the game right after install with no tweaks. Happening on shadow of mordor for sure. Unsure if there should be specific default setting for verified games. Thank you for your time

**Comment by NGnius** [comment]
Is this related to https://github.com/NGnius/PowerTools/issues/39#issuecomment-1255625229 ?

**Comment by NGnius** [comment]
I'll take that as "No". Can you confirm for me that you haven't set custom PowerTools settings for Shadow of Mordor or others that are crashing? Also have you set default settings?

**Comment by joshuadavidthomas** [comment]
Just to add, I am having a similar issue with Elden Ring. 

I would try to launch the game, it would get through loading the anti-cheat stuff, then get stuck on a white screen. There was an earlier issue with that game and this happening that looked like it was fixed on more recent versions of Proton, but even with the newer version it was still crashing for me. 

I was initially on the beta version of SteamOS, but downgraded to stable as part of my effort to get the game working. I tried various versions of Proton, including the latest GET version, but what got it working finally was uninstalling PowerTools. So while it may be related to https://github.com/NGnius/PowerTools/issues/39#issuecomment-1255625229, it does seem to be a different issue.

If needed, I can provide further details.

**Comment by BrooklynTheDogg** [comment]
Ok so i ran some test scenarios. First thing first i guess. Uninstalling powertools leaves that folder you mentioned in .config behind i cant remove it. 
In that folder there is a json file for shadow of mordor. I added whats in it to this post. 
![20221004_101313](https://user-images.githubusercontent.com/99148920/193844346-196653da-6dde-436b-a4bc-f2206e2e372a.jpg)
![20221004_101303](https://user-images.githubusercontent.com/99148920/193844373-a988d72f-fb93-4b1f-9736-fc7f751ca7ac.jpg)

Once i install powertools it blocks sow from running. Uninstall works fine. I havent made any settings changes. I tried clicking default settings no difference. I turned off persistant as well. 
Side note:
I was able to clear out to sow json file and save it.
Then rebooting and installing powertools. The game loads properly. At this point its beyond my experience to what the catch is. Hopefully someone can provide some insight. Thank you!

**Comment by NGnius** [comment]
Looks like there's at least 1 bug in PowerTools there: non-persistent configs should be deleted, not saved. You should never see a json file with persistent set to false. I'll try to sort out why that happens. I thought I fixed that in a recent release, but apparently not well enough.

As to why that config is crashing a game, I can only guess that SOW does not like CPU threads changing as it's starting up (the config looks like SMT was disabled).

@joshuadavidthomas could you check that same folder for an ER config? I just want to confirm it's the same issue

**Comment by joshuadavidthomas** [comment]
@NGnius Yep, I have a handful of JSON files in `~/.config/powertools`: some related to specific games and a `default_settings.json` that does have the `persistent` key set to `false`.

**Comment by BrooklynTheDogg** [comment]
Are those supposed to stay after uninstal? I had only used powertools to tweak cores when using dolphin never sow. Perhaps a secondary option to delete all the game json configs on uninstall, like a check box?

**Comment by NGnius** [comment]
Can y'all let me know if the v1.0.5 update fixes this? The update should be available on the store once this is sorted https://github.com/SteamDeckHomebrew/decky-plugin-database/pull/93

**Comment by BrooklynTheDogg** [comment]
Just checked not there yet. I will give it a shot once it goes live.

**Comment by NGnius** [comment]
Should be on the store now

**Comment by BrooklynTheDogg** [comment]
Ok so i gave it a try but and this is a problem i need to solve first. When i was helping troubleshoot the way i figured it out was to wipe out the contents of a sow file so now its blank. When i try to make to make changes wth the 105 version it doesnt seem to be writing into that file. Do you have a command line i can run to delete that file and start fresh since i dont have the optionn in the gui?

**Comment by NGnius** [comment]
You wipe out all PowerTools configs with `sudo rm -rf ~/.config/powertools`.
Or you can remove a specific file with `sudo rm <file>`

**Comment by NGnius** [comment]
Just noticed you also started https://github.com/SteamDeckHomebrew/decky-loader/issues/195

If you can confirm that that actually fixes it, I'll close this.

**Comment by BrooklynTheDogg** [comment]
Tomorrow or monday ill be able to give it the time to test for you if that works.

**Comment by BrooklynTheDogg** [comment]
Finally tested and appears to be working!! Big thank you!
